nucypher/utilities/metrics.py

```diff
@@ -26,40 +77,223 @@ def collect_prometheus_metrics(ursula):
+        "token_balance_gauge": Gauge(f'{metrics_prefix}_staker_token_balance', 'NuNit balance'),
+        "worker_eth_balance_gauge": Gauge(f'{metrics_prefix}_worker_eth_balance', 'Worker Ethereum balance'),
+        "worker_token_balance_gauge": Gauge(f'{metrics_prefix}_worker_token_balance', 'Worker NuNit balance'),
+        "requests_counter": Counter(f'{metrics_prefix}_http_failures', 'HTTP Failures', ['method', 'endpoint']),
+        "host_info": Info(f'{metrics_prefix}_host_info', 'Description of info'),
+        "active_stake_gauge": Gauge(f'{metrics_prefix}_active_stake', 'Active stake'),
+        "owned_tokens_gauge": Gauge(f'{metrics_prefix}_owned_tokens', 'All tokens that belong to the staker, including '
+                                                                      'locked, unlocked and rewards'),
+        "unlocked_tokens_gauge": Gauge(f'{metrics_prefix}_unlocked_tokens', 'Amount of unlocked tokens'),
+        "available_refund_gauge": Gauge(f'{metrics_prefix}_available_refund', 'Available refund'),
+        "policies_held_gauge": Gauge(f'{metrics_prefix}_policies_held', 'Policies held'),
+        "current_period_gauge": Gauge(f'{metrics_prefix}_current_period', 'Current period'),
+        "current_eth_block_number": Gauge(f'{metrics_prefix}_current_eth_block_number', 'Current Ethereum block'),
+        "substakes_count_gauge": Gauge(f'{metrics_prefix}_substakes_count', 'Substakes count'),
+        "current_worker_is_me_gauge": Gauge(f'{metrics_prefix}_current_worker_is_me', 'Current worker is me'),
+        "worklock_deposited_eth_gauge": Gauge(f'{metrics_prefix}_worklock_current_deposited_eth',
+                                              'Worklock deposited ETH'),
+        "worklock_remaining_work_gauge": Gauge(f'{metrics_prefix}_worklock_refund_remaining_work',
+                                               'Worklock remaining work'),
+        "worklock_refund_completed_work_gauge": Gauge(f'{metrics_prefix}_worklock_refund_completedWork',
+                                                       'Worklock completed work'),
+    }
+
+    event_metrics_collectors = get_event_metrics_collectors(ursula, metrics_prefix)
+
+    if not ursula.federated_only:
+        staking_agent = ContractAgency.get_agent(StakingEscrowAgent, registry=ursula.registry)
+
+        node_metrics["current_worker_is_me_gauge"].set(
+            staking_agent.get_worker_from_staker(ursula.checksum_address) == ursula.worker_address)
 
     # Scheduling
-    metrics_task = task.LoopingCall(collect_prometheus_metrics, ursula=ursula)
+    metrics_task = task.LoopingCall(collect_prometheus_metrics, ursula=ursula,
+                                    event_metrics_collectors=event_metrics_collectors, node_metrics=node_metrics)
     metrics_task.start(interval=10, now=False)  # TODO: make configurable
 
     # WSGI Service
     root = Resource()
     root.putChild(b'metrics', MetricsResource())
+    root.putChild(b'json_metrics', JSONMetricsResource())
```

Comment:
Are these two intended to be separate metrics services?

Reply:
This way was more comfortable for devops.

Reply:
Hmm... not sure I understand exactly what is happening here.

Reply:
`/metrics` endpoint - for Prometheus (prometehus response format)
`/json_metrics` - for  any other service (JSON)